### ISSUE TITLE
Feature request: run multiple train jobs in parallel

### ISSUE THREAD

**Issue by yartem** [neuromation/neuro-flow#12]
Currently, Makefile can run multiple train jobs in parallel:
```
$ make train RUN=experiment-1
$ make train RUN=experiment-2
```

Also, the guys use this to create new targets, which, I guess, is already easy to implement with current neuro-flow:
https://github.com/neuromation/mask-detector/blob/41a2257d715ba2a1bb83aabea901bebb3bd09c46/Makefile#L426-L441

**Comment by asvetlov** [comment]
We decided to support `multiple=true` yaml parameter for such jobs.
To distinguish job instances use a suffix. The suffix can be provided by a user with `--suff my-suffix`, otherwise a short random string is autogenerated.
`neuro-flow ps` shows all running jobs with their suffixes.
Implementation detail:  `real_id` should be used instead of `id` in `JobCtx`.

**Comment by asvetlov** [comment]
Fixed by #25
